app/src/main/java/com/chartiq/chartiqsample/studies/StudyOptionsActivity.java

```diff
@@ -261,7 +261,29 @@ public void onClick(View v) {
         if ("auto".equals(parameter.color)) {
             color.setBackgroundColor(Color.BLACK);
         } else {
-            color.setBackgroundColor(Color.parseColor(parameter.color));
+            try{
+                if(parameter.color.contains("rgb")) {
+                    String subString = parameter.color.substring(parameter.color.indexOf('(') + 1, parameter.color.indexOf(')'));
+                    String rgbColors[] = subString.split(",");
+                    int parsedColor = 0;
+                    int alphaValue = 255;
+
+                    // contains an alpha value
+                    if(rgbColors.length == 4) {
+                        double value = Double.parseDouble(rgbColors[3].trim());
+                        value = Math.floor(value >= 1.0 ? 255 : value * 256.0); // use 256 for floating point precision when value is less than 1.0
```

Comment:
Was this protection necessary?  I've never seen rgba with alpha above 1.  That's like an error.